docs/en/guide/configurations.md

```diff
@@ -142,54 +145,237 @@ globalConfig:
       terminationGracePeriodSeconds: 60
 ```
 
-### Inherit from CSI Node (deprecated) {#inherit-from-csi-node}
+</details>
 
-:::tip
-Starting from v0.24, CSI Driver can customize mount pods and sidecar containers in the [ConfigMap](#configmap), legacy method introduced in this section is not recommended.
-:::
+All supported configuration items and PVC selectors are listed in the above snippet, but the behavior of each item is introduced in below sections, please read before use.
 
-Mount pod specs are mostly inherited from CSI Node, for example if you need to enable `hostNetwork` for mount pods, you have to instead add the config to CSI Node:
+### Custom mount image {#custom-image}
 
-```yaml title="values-mycluster.yaml"
-node:
-  hostNetwork: true
+#### Via Configmap
+
+Minimum required version is CSI Driver v0.24.0. Upon modification, application pods or mount pods need to be re-created for changes to take effect. If you decide to re-create mount pod, be sure to enable [automatic mount point recovery](./configurations.md#automatic-mount-point-recovery) in advance, to avoid permanent loss of mount point within the application pod.
```

Comment:
```suggestion
The minimum required version is CSI Driver v0.24.0. Upon modification, application pods or Mount Pods need to be re-created for changes to take effect. If you decide to re-create Mount Pods, be sure to enable [automatic mount point recovery](./configurations.md#automatic-mount-point-recovery) in advance, to avoid permanent loss of mount point within the application pod.
```